helm-charts/HPA.md

```diff
@@ -100,60 +100,37 @@ used model, data type and image versions, the specified resource values may need
 
 ### Post-install
 
-Above step created custom metrics config for Prometheus-adapter suitable for HPA use.
+Above step created custom metrics config for Prometheus-adapter, suitable for HPA use.
 
-Take backup of existing custom metrics config before replacing it:
+However, Helm does not allow OPEA chart to overwrite Prometheus-adapter configMap, as
+it belongs to another chart. Therefore manual step is needed to overwrite its current
```

Comment:
Therefore, a manual step is needed to overwrite its current custom metric rules.